src/client/components/app-footer/app-footer.vue

```diff
@@ -1,37 +1,24 @@
-<template>
+  <template>
   <footer role="contentinfo" class="app-footer">
     <div class="app-footer__container">
       <div>
         <nuxt-link :to="localeUrl('index')" class="app-footer__identity">
           <img class="app-footer__logo" src="/images/logo.svg" alt="" width="32" height="40" />
           <h1 class="app-footer__title">{{ appConfig.title }}</h1>
         </nuxt-link>
-        <div class="app-footer__social">
-          <a class="app-footer__social-link" v-if="social.facebook" :href="social.facebook" @click="track(social.facebook)">
-            <facebook-icon />
-            <span class="a11y-sr-only">Facebook</span>
-          </a>
-          <a class="app-footer__social-link" v-if="social.twitter" :href="social.twitter" @click="track(social.twitter)">
-            <twitter-icon />
-            <span class="a11y-sr-only">Twitter</span>
-          </a>
-          <a class="app-footer__social-link" v-if="social.googlePlus" :href="social.googlePlus" @click="track(social.googlePlus)">
-            <google-plus-icon />
-            <span class="a11y-sr-only">Google Plus</span>
-          </a>
-          <a class="app-footer__social-link" v-if="social.instagram" :href="social.instagram" @click="track(social.instagram)">
-            <instagram-icon />
-            <span class="a11y-sr-only">Instagram</span>
-          </a>
-          <a class="app-footer__social-link" v-if="social.youtube" :href="social.youtube" @click="track(social.youtube)">
-            <youtube-icon />
-            <span class="a11y-sr-only">YouTube</span>
-          </a>
-          <a class="app-footer__social-link" v-if="social.linkedin" :href="social.linkedin" @click="track(social.linkedin)">
-            <linkedin-icon />
-            <span class="a11y-sr-only">LinkedIn</span>
-          </a>
-        </div>
+
+      <ul class="app-footer__social">
+        <li
+          v-for="link in appConfig.socialLinks"
+          :key="link.id"
```

Comment:
`link.id` is not defined. Either `pick` the IDs in the Dato dump or use `index` here.

Reply:
I picked the `id` in the Dato dump.